New package: Fibers v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: Fibers
- Repository: https://github.com/lincbrain/Fibers.jl
- Created by: @ayendiki
- Version: v1.0.0
- Commit: 06569a586754855322f011482cbf9bb3a5021b3b
- Reviewed by: @ayendiki
- Reference: https://github.com/lincbrain/Fibers.jl/issues/17#issuecomment-2076267056
- Description: Julia package for reconstructing fiber trajectories from multi-scale, multi-modal imaging data
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598d394b201bb2a1d04fb09d4daa7dd43cf4e96d47101297b84849109d489f5dbf741663ccedbd71e1c8ffb55b12ce5f08d2331628d668aa94ed1c48f9fff727f20de6eaac1b4309e56256ce6c945d73af924e5cb3288ff29899d5ec8d41f5e9bcfbff7d89458a47fbd3f0b140b22eae1d87c8c2df8307dab5cb76e17e291df878b -->

goerz commented:
[noblock] name similarity is okay, but the other issues will have to be addressed

JuliaRegistrator committed:
New package: Fibers v1.0.0

UUID: 79c5424f-e3fe-4f6a-a484-e0e793a8858c
Repo: https://github.com/lincbrain/Fibers.jl.git
Tree: 933bc901bf8be096527127a45f9d316933f5deed

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
